crates/core/src/network/partition_processor_rpc_client.rs

```diff
@@ -335,8 +337,18 @@ where
                     inner: inner_request,
                 },
             )
-            .await?;
+            .await?
+            .into_body();
+
+        if rpc_result.is_err() && rpc_result.as_ref().unwrap_err().likely_stale_route() {
+            trace!(
```

Comment:
Not debug as this is proportional to ingress traffic, which could be very high volume.